content/master/guides/composition-functions.md

```diff
@@ -0,0 +1,1002 @@
+---
+title: Composition Functions
+state: alpha
+---
+
+Composition Functions allow you to supplement or replace your Compositions with
+advanced logic. You can build a Function using general purpose programming
+languages such as Go or Python, or relevant tools such as Helm, kustomize, or
+CUE. Functions compliment contemporary "Patch and Transform" (P&T) style
+Composition - it's possible to use only P&T, only Functions, or a mix of both in
+the same Composition.
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: example
+spec:
+  compositeTypeRef:
+    apiVersion: database.example.org/v1alpha1
+    kind: XPostgreSQLInstance
+  functions:
+  - name: my-cool-Function
+    type: Container
+    container:
+      image: xpkg.upbound.io/my-cool-Function:0.1.0
+```
+
+A Composition Function is a short-lived OCI container that tells Crossplane how
+to reconcile a Composite Resource (XR). The preceding example shows a minimal
+`Composition` that uses a Composition Function. Note that it has a `functions`
+array rather than the typical P&T style array of `resources`.
+
+## Enabling Functions
+
+To enable support for Composition Functions you must:
+
+ * Enable the alpha feature flag in Crossplane.
+ * Deploy a runner that is responsible for running Functions.
+
+```console
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm repo update
+
+# Find the latest RC build of Crossplane - e.g. 1.11.0-rc.0.274.g51a07e50
+helm search repo crossplane-master/crossplane --devel
+
+RC_VERSION="v1.11.0-rc.0.274.g51a07e50"
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane \
+    --devel --version ${RC_VERSION} \
+    --set "args={--debug,--enable-composition-Functions}" \
+    --set "xfn.enabled=true" \
+    --set "xfn.args={--debug}"
+```
+
+The preceding Helm command installs Crossplane with the Composition Functions
+feature flag enabled, and with the reference "`xfn`" Composition Function runner
```

Comment:
The quotes here are because I'm introducing a name. Would they make more sense if I dropped the backticks here and rephrased to "the reference Composition Function runner, named "xfn""?

https://www.grammarly.com/blog/quotation-marks-around-a-single-word/